### DISCUSSION TITLE
green point guilde

### DISCUSSION THREAD

**Discussion by Faith1301** [fedorovvl/tso_client#266] [Q&A]
hello i don't see the green points (connected members of the guilde)

**Comment by fedorovvl** [comment]
hi. online status depends on chat server.. if smth wrong with chat server green dots not shown

**Reply by Faith1301** [reply]
it's strange it's only on the guilde  not thefriends

**Comment by iraxxxx** [comment]
We have the same problem with the guild member's green activity points. They died over 8 months ago. Active guild players are visible in the friends tab, while in the guild tab all lights are off.
